src/app-layout/drawer/resizable-drawer.tsx

```diff
@@ -18,33 +18,23 @@ import { TOOLS_DRAWER_ID } from '../utils/use-drawers';
 
 export const ResizableDrawer = ({
   onResize,
-  size,
-  minSize,
-  getMaxWidth,
+  maxWidth,
+  minWidth,
   refs,
   activeDrawer,
   toolsContent,
   ...props
 }: ResizableDrawerProps) => {
-  const { isOpen, children, isMobile } = props;
-  const [relativeSize, setRelativeSize] = useState(0);
+  const { isOpen, children, width, isMobile } = props;
 
-  useEffect(() => {
```

Comment:
Main subject of this change. Remove `useEffect` calls, make the layout calculated from props directly